test/pytest/service-desktop/test_device_info.py

```diff
@@ -1,32 +1,44 @@
-# Copyright (c) 2017-2021, Mudita Sp. z.o.o. All rights reserved.
+# Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
 # For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
 import pytest
-from harness.api.device_info import GetDeviceInfo
-from harness.request import TransactionError
+import re
+from harness.api.device_info import GetDeviceInfo, GetDiagnosticFilesList, DiagnosticsFileList
+
 
 @pytest.mark.service_desktop_test
 @pytest.mark.usefixtures("phone_unlocked")
-def test_device_info(harness):
+def test_get_device_information(harness):
     ret = GetDeviceInfo().run(harness)
 
     assert int(ret.diag_info["batteryLevel"]) <= 100
-    assert ret.diag_info["batteryState"] is not None
-    assert ret.diag_info["selectedSim"] is not None
-    assert ret.diag_info["trayState"] is not None
-    assert ret.diag_info["signalStrength"] is not None
-    assert ret.diag_info["accessTechnology"] is not None
-    assert ret.diag_info["networkStatus"] is not None
+    assert ret.diag_info["batteryState"] in ['1', '2']
+    assert ret.diag_info["selectedSim"] in ['0', '1', '4']
+    assert ret.diag_info["trayState"] == '1'
+    assert 0 <= int(ret.diag_info["signalStrength"]) < 5
+    assert ret.diag_info["accessTechnology"] in ['0', '2', '3', '4', '5', '6', '7', '255']
+    assert 0 <= int(ret.diag_info["networkStatus"]) < 7
     assert ret.diag_info["networkOperatorName"] is not None
-    assert ret.diag_info["fsTotal"] is not None
-    assert ret.diag_info["fsFree"] is not None
+    assert 0 < int(ret.diag_info["fsTotal"]) < 16000
+    assert 0 < int(ret.diag_info["fsFree"]) < 16000
     assert 0 < int(ret.diag_info["fsFreePercent"]) <= 100
-    assert ret.diag_info["gitRevision"] is not None
-    assert ret.diag_info["gitTag"] is not None
+    assert re.match("^(\d|[a-z]){8}$", ret.diag_info["gitRevision"])
     assert ret.diag_info["gitBranch"] is not None
-    #assert ret.diag_info["updateHistory"] is not None
-    assert ret.diag_info["currentRTCTime"] is not None
-    assert ret.diag_info["version"] is not None
-    assert ret.diag_info["serialNumber"] is not None
-    assert ret.diag_info["caseColour"] is not None
-    assert ret.diag_info["backupLocation"] is not None
+    assert int(ret.diag_info["currentRTCTime"]) > 1641991996
+    assert re.match("^\d\.\d\.\d", ret.diag_info["version"])
+    assert re.match("^\d{14}$", ret.diag_info["serialNumber"])
+    assert ret.diag_info["caseColour"] in ["gray", "black"]
+    assert ret.diag_info["backupLocation"] == "/sys/user/backup"
+
+
+@pytest.mark.service_desktop_test
+@pytest.mark.usefixtures("phone_unlocked")
+def test_get_list_of_log_files(harness):
+    ret = GetDiagnosticFilesList(DiagnosticsFileList.LOGS).run(harness)
+    assert "/sys/user/logs/MuditaOS.log" in ret.files
 
+# Deactivated until we have possibility to generate crash dumps on demand [CP-690]
```

Comment:
just remove? or if you want to deactivate - then deactivate with pytest function :)

Reply:
Removed

Reply:
I've changed my mind.
Restored, and marked with `@pytest.mark.skip`